eZ/Publish/Core/Base/Container/Compiler/HelperFieldTypeRegistryPass.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\Core\Base\Container\Compiler;
+
+use eZ\Publish\Core\Repository\Helper\FieldTypeRegistry;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use LogicException;
+use Symfony\Component\DependencyInjection\Reference;
+
+class HelperFieldTypeRegistryPass implements CompilerPassInterface
```

Comment:
Why do we need both `HelperFieldTypeRegistryPass` and `FieldTypeRegistryPass`? There's a lot of duplicated code.

Reply:
I agree with @alongosz here.